migrate/chaindb4to5.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
```

Comment:
Should we have an explicit note somewhere that this basically just removes your old db and does a fresh sync with the new indexing?

Reply:
Yeah we can add that to the `CHANGELOG`

Reply:
Added to `CHANGELOG`